modules/skybrowser/skybrowsermodule_lua.inl

```diff
@@ -765,6 +765,17 @@ namespace {
         );
 }
 
+/**
+ * Stop animations
+ * \param identifier Identifier of the sky browser
+ */
+[[codegen::luawrap]] void stopAnimations(std::string identifier) {
+    using namespace openspace;
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+    TargetBrowserPair* pair = module->getPair(identifier);
```

Comment:
For consistency with the naming scheme, we should rename `getPair` to just `pair`.  We're only using the `get` variant if the function takes a reference to the object being filled, which is not something that we'd want to do everywhere anyway. Instead:
```cpp
TargetBrowserPair* pair = module->pair(identifier);
```